docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
     restart: always
     environment:
       API_URL: http://web:3001
+      NEXTGEN_API_URL: http://web:8008
```

Comment:
I don't know if we're looking to set this up in deployed environments at this time, but we'll eventually need to add vars in each github env in this, and configure our github actions to pick up the variables and set them with happy config https://github.com/chanzuckerberg/czid-graphql-federation-server/blob/9af53fe5d9b8c0f88659014e054c430f708e1cfd/.github/workflows/deploy-sandbox.yml#L34-L39

Reply:
Yeah, it makes sense to hold off deploying it but I added it here so that it is available for local development.

Reply:
Yea, we do have to have this for local dev regardless, but my comment pertained to deployed environments.  for dev and sandbox, #96 added these env vars as part of the TF config